lib/std/io/change_detection_stream.zig

```diff
@@ -0,0 +1,58 @@
+const std = @import("../std.zig");
+const io = std.io;
+const mem = std.mem;
+const assert = std.debug.assert;
+
+pub fn ChangeDetectionStream(comptime OutStreamType: type) type {
```

Comment:
Could you add a doccomment of what this does?

Reply:
Sure can